core/readme/package.json

```
@@ -32,7 +32,7 @@
   "dependencies": {
     "dompurify": "^2.2.6",
     "jsdom": "15.2.1",
-    "marked": "2.1.3"
+    "marked": "4.0.10"
```

```suggestion
    "marked": "^4.0.10"
```
Won't be a range here better to allow future patch updates without release?


I think they strictly follow semver

---

I always pin personal rule, it saved me of many headaches, like the `colors.js` one or any other external  mistake. Not big fan of `^`. `` ^_^`

---

Ok. 🤗